recipe/meta.yaml

```
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
```

Suggesting switching to hashed URL while the upstream PyPI issue with the redirect is worked out

```suggestion
  url: https://files.pythonhosted.org/packages/57/f6/bb2532524de2bad548d5a0d731659389dc9fb3483b18715fa3c359c4111c/distributed-{{ version }}.tar.gz
```